ai_employee_decision_system/api/app.py

```
@@ -430,17 +430,68 @@ def process_query(
     if not query:
         raise HTTPException(status_code=400, detail="Query is required")
     
-    result = ai_service.process_query(query, context)
+    result = ai_service.process_query(query, context=context, use_rag=use_rag, top_k=top_k)
     return result
 
 
+@app.post("/ai/search", tags=["AI"])
+def search_knowledge_base(
+    request: Dict[str, Any],
+    current_user: User = Depends(get_current_user),
```

Ruff (0.13.1)

Unused function argument: `current_user`

(ARG001)